xchange-stream-gemini/src/main/java/info/bitrich/xchangestream/gemini/GeminiStreamingMarketDataService.java

```diff
@@ -57,7 +57,7 @@ private boolean filterEventsByReason(JsonNode message, String type, String reaso
   @Override
   public Observable<OrderBook> getOrderBook(CurrencyPair currencyPair, Object... args) {
 
-    Integer maxDepth = args.length > 0 ? (Integer) args[0] : null;
+    int maxDepth = args.length > 0 ? (int) args[0] : 0;
```

Comment:
args[0] could be null. How about something like

```
int maxDepth = (int) MoreObjects.firstNonNull(args.length > 0 ? args[0] : null, 0);
```

Reply:
Yes, makes sense. I will push the changes in a minute.